biz.aQute.launcher/src/aQute/launcher/Launcher.java

```diff
@@ -1238,16 +1267,8 @@ private void row(PrintStream out, Object... parms) {
 		out.println();
 	}
 
-	String toDate(long t) {
-		Calendar c = Calendar.getInstance();
-		c.setTimeInMillis(t);
-		return fill(c.get(Calendar.YEAR), 4) + fill(c.get(Calendar.MONTH), 2) + fill(c.get(Calendar.DAY_OF_MONTH), 2)
-			+ fill(c.get(Calendar.HOUR_OF_DAY), 2) + fill(c.get(Calendar.MINUTE), 2);
-	}
-
-	private String fill(int n, int width) {
-		return fill(Integer.toString(n), width, '0', -1);
-	}
+	static final DateTimeFormatter FORMAT = DateTimeFormatter.ofPattern("YYYYMMddHHmm")
```

Comment:
I see the zone was already in there but I would MUCH prefer to use UTC always in bnd.

Reply:
For storing values or for computation, I agree. But for displaying values, presumably the intent is for it to be human intelligible? In which case, wouldn't the system default time done be more useful? If the user really wanted to see values in UTC presumably they would have UTC set as their system default time zone?

If you still disagree, this is an easy change.

Reply:
By way of comparison, file timestamps in Linux are stored in UTC but when you do an ls they are displayed in system time zone (from memory - been a while since I used Linux).

Reply:
I think UTC is better as the output is consistent and does not vary based upon the location of the system. This is nice when your CI system is in the cloud and you don't care what time zone it is in. You only care that your output is consistent.

Reply:
Done.